docs/source/user-guide/index.rst

```diff
@@ -2,10 +2,42 @@
 User guide
 **********
 
+Welcome to the conda build user guide. Here you can find
+tutorials and recipes as well as information about environment
+variables and wheel files.
+
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
+   getting-started
    tutorials/index
    recipes/index
    environment-variables
    wheel-files
+
+
+**Tutorials**
+
+The :doc:`tutorials <../user-guide/tutorials/index>` will guide you through
+how to build conda packages---whether you're creating a simple
+package without compilers, using conda skeleton, creating from
+scratch, or building R packages using skeleton CRAN.
+
+**Recipes**
+
+Conda build uses :doc:`recipes <../user-guide/recipes/index>` to create conda packages.
+We have guides on how to build a package without a recipe and
+debugging conda recipes, as well as sample recipes for you to use.
+
+**Environment variables**
+
+Use our :doc:`environment variables <environment-variables>` guide
+to set variables and see what environment variables affect processes.
```

Comment:
You aren't really using the guide to set variables, so consider rewording slightly, to "...to understand which environment variables are available, set, and inherited, and how they affect different processes."